webrender/src/platform/windows/font.rs

```diff
@@ -186,9 +188,14 @@ impl FontContext {
                                                     dwrite_measure_mode,
                                                     options);
 
-        // XX use the xform to handle subpixel positioning (what skia does), I believe that keeps
-        //let xform = dwrote::DWRITE_MATRIX { m11: 1.0, m12: 0.0, m21: 0.0, m22: 1.0, dx: 0.0, dy: 0.0 };
-        dwrote::GlyphRunAnalysis::create(&glyph_run, 1.0, None,
+        let x_offset: f32 = Into::<f64>::into(x_suboffset) as f32;
```

Comment:
This seems a bit over complicated - perhaps convert it to a local f64 and then cast to f32 when creating the matrix or something like that. Can be done as a follow up anyway, it's not major.